liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -109,26 +109,42 @@ private static void initializeAppsLocation(File serverXML, File configDir, File
             // get variables from server.env 
             File cfgDirFile = getFileFromConfigDirectory("server.env");
             Properties propServerEnv = new Properties();
+            FileInputStream fis = null;
+
             if (cfgDirFile != null) {
-                propServerEnv.load(new FileInputStream(cfgDirFile));
+                fis = new FileInputStream(cfgDirFile);
+                propServerEnv.load(fis);
                 props.putAll(propServerEnv);
             } else if (serverEnvFile.exists()) {
-                propServerEnv.load(new FileInputStream(serverEnvFile));
+                fis = new FileInputStream(serverEnvFile);
+                propServerEnv.load(fis);
                 props.putAll(propServerEnv);
             }
             
+            if (fis != null) {
```

Comment:
Shouldn't you close it in a try-finally block, and would it be better to have a separate method to load a properties object from a inputStream.